### ISSUE TITLE
LUIS CLI - Does not work behind corporate proxy - SSL23_GET_SERVER_HELLO:unknown protocol

### ISSUE THREAD

**Issue by martinoss** [microsoft/botbuilder-tools#912]
## Tool
Name: **LUIS**
Version: **2.2.2**
OS: **Windows 10**

## Describe the bug
While trying to setup the Virtual Assistant Solution (https://github.com/Microsoft/AI/), the deployment failed with this error:

```
Updating en-us deployment scripts ...
Generating en-us LUIS and QnA Maker models from .lu files ...
Deploying en-us resources...
Command failed: luis import application --region westeurope --appName "XXXen_General" --in C:\Code\Prototyping\AI\solutions\Virtual-Assistant\src\csharp\assistant\DeploymentScripts\en\general.luis --authoringKey XXX --msbotwrite 
EPROTO 101057795:error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol:openssl\ssl\s23_clnt.c:827:
```

I'm pretty sure it is a problem caused by having a corporate proxy server.
I have several settings and environment variables in place (for different applications like git, npm, azure cli, azure storage queue, bower, ...)  to point to a local proxy server (Fiddler) that also can handle NTLM authentication.

Environment variables `HTTPS_PROXY=http://127.0.0.1:8888` and 
`HTTP_PROXY=http://127.0.0.1:8888` seems to be respected somehow - without I get a timeout exception. For `HTTPS_PROXY`, the value is intendedly without an `s` and needs to be without it. This could also be the root of the problem if some validation or incorrect handling takes place down the tool chain.

Probably it is related to this:
https://github.com/Azure/ms-rest-js/issues/169

## To Reproduce
It should be reproducable when using Telerik Fiddler as a local proxy and setting the required environment variables respected by many tools and libaries (HTTPS_PROXY, HTTP_PROXY).

## Expected behavior
There should be documentation about configuring luis to use in conjunction with proxy servers according to the dependencies used by LUIS (ms-rest-js, request, axios?)

## Screenshots
--

## Additional context
--

[bug]

**Comment by justinwilaby** [comment]
@sgellock - Looks like [ms-rest-js merged the fix](https://github.com/Azure/ms-rest-js/pull/315) for proxy support. We need to increment the ms-rest-js version on our tooling to pick this up once they release

**Comment by sgellock** [comment]
Sounds like a 4.4 item. Yes?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by a-hohaze** [comment]
Hey thanks for the info, the label is pumped, QNAMaker CLI tool seems like doesn't use ms-rest, why is ms-rest used here?

**Comment by sgellock** [comment]
@a-hohaze not sure what you question is, so I'll take a shot at an answer.  For consistency across our toolchain we'd like our clients to standardize on ms-rest

**Comment by joalmeid** [comment]
Hello Fuse team.
Any updates on this? 

After the [ms-rest-js fix](https://github.com/Azure/ms-rest-js/pull/315) merged, can we have an ETA for LUIS CLI?

**Comment by sgellock** [comment]
@cahann is the correct person to speak to this.  Carol?

**Comment by gabog** [comment]
Hey @cahann, can you please chime into the thread?

Thanks

**Comment by a-hohaze** [comment]
the ms-rest-js version was updated with PR #1004 which includes a fix for this issue.

**Comment by cleemullins** [comment]
Closed by #1116
